app/main.dev.js

```diff
@@ -70,7 +70,10 @@ app.on('ready', async () => {
   mainWindow = new BrowserWindow({
     show: false,
     width: 1024,
-    height: 728
+    height: 728,
+    webPreferences: {
+      nodeIntegration: true
+    }
```

Comment:
Will this "works", it's strongly discouraged by the electron docs, read more here: https://electronjs.org/docs/tutorial/security